docs/prompt_engineering/how_to_guides/index.md

```diff
@@ -7,6 +7,7 @@ Step-by-step guides that cover key tasks and operations for doing prompt enginee
 Organize and manage prompts in LangSmith to streamline your LLM development workflow.
 
 - [Create a prompt](./how_to_guides/create_a_prompt)
+- [Include multimodal content](./how_to_guides/multimodal_content)
```

Comment:
imo this should be in the playground section since its not specific to prompt hub